src/main/java/org/peergos/HostBuilder.java

```diff
@@ -89,6 +85,13 @@ public HostBuilder listen(List<MultiAddress> listenAddrs) {
         return this;
     }
 
+
+    public HostBuilder listenLocalhost(int listenPort) {
+        listenAddrs.add("/ip4/127.0.0.1/tcp/" + listenPort);
+        return this;
+    }
```

Comment:
We should probably use the new approach for adding localhost address. Otherwise we won't inherit any future changes.

Inside `initializeProtocols`:
```
        HostBuilder hostBuilder = new HostBuilder()
                .generateIdentity()
                .listen(List.of(new MultiAddress("/ip4/0.0.0.0/tcp/" + HOST_PORT)));
```

Reply:
Shouldn't it be .listen(List.of(new MultiAddress("/ip4/127.0.0.1/tcp/" + HOST_PORT))); or we are fine with binding to all interfaces (global ip - 0.0.0.0)?